src/Paket.Core/ProjectFile.fs

```diff
@@ -129,8 +129,6 @@ type ProjectFile =
             node.ParentNode.RemoveChild(node) |> ignore
 
     member this.UpdateFileItems(fileItems : FileItem list, hard) = 
-        this.DeletePaketNodes("Compile")
```

Comment:
Humm, So the `InstallProcess.InstallIntoProjects` also removes stuff? I guess by virtue of the fact that if it isn't included in that file list then it shouldn't be there? Correct?